Expensify/App
# Update icons to use $ icon

Please review @puneetlath cc @Expensify/design 

### Details
Updates the icon asset for all clients:

1. sign in page
2. chat switcher
3. app icon
4. favicon
5. Push notification asset on Android
6. README.md

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/148907

### Tests
1. Run `npm run ios` -> Verify icon is updated
2. Run `npm run android` -> Verify icon is updated
3. Run `npm run web` -> Verify icon is updated
4. Run `npm run desktop-build` -> verify icons are updated

### Screenshots
##### Desktop
<img width="652" alt="Screen Shot 2020-12-22 at 3 37 02 PM" src="https://user-images.githubusercontent.com/2838819/102943213-49307000-446c-11eb-8868-434f09d7dd19.png">
<img width="1164" alt="Screen Shot 2020-12-22 at 3 37 13 PM" src="https://user-images.githubusercontent.com/2838819/102943216-4afa3380-446c-11eb-9774-c4cc4d1ba7f5.png">
<img width="1312" alt="Screen Shot 2020-12-22 at 3 37 30 PM" src="https://user-images.githubusercontent.com/2838819/102943217-4b92ca00-446c-11eb-85ff-58d77430e012.png">
<img width="396" alt="Screen Shot 2020-12-22 at 3 37 35 PM" src="https://user-images.githubusercontent.com/2838819/102943218-4cc3f700-446c-11eb-9a63-8c0e611e5d32.png">

##### Android
<img width="487" alt="Screen Shot 2020-12-22 at 3 17 32 PM" src="https://user-images.githubusercontent.com/2838819/102943233-58afb900-446c-11eb-872c-6b20f4151b55.png">
<img width="487" alt="Screen Shot 2020-12-22 at 3 17 17 PM" src="https://user-images.githubusercontent.com/2838819/102943234-59e0e600-446c-11eb-869b-b1e1ec054454.png">
<img width="487" alt="Screen Shot 2020-12-22 at 2 58 11 PM" src="https://user-images.githubusercontent.com/2838819/102943235-5b121300-446c-11eb-8edf-6658f9f9bc37.png">

##### iOS
<img width="544" alt="Screen Shot 2020-12-22 at 2 44 46 PM" src="https://user-images.githubusercontent.com/2838819/102943278-72510080-446c-11eb-876e-d85b25afcb62.png">

##### Web
<img width="502" alt="Screen Shot 2020-12-22 at 2 42 38 PM" src="https://user-images.githubusercontent.com/2838819/102943296-7b41d200-446c-11eb-87a7-6424503cb01a.png">
<img width="307" alt="Screen Shot 2020-12-22 at 2 42 22 PM" src="https://user-images.githubusercontent.com/2838819/102943298-7c72ff00-446c-11eb-8622-5649e894f473.png">
<img width="397" alt="Screen Shot 2020-12-22 at 2 28 36 PM" src="https://user-images.githubusercontent.com/2838819/102943299-7d0b9580-446c-11eb-9902-a568332547b0.png">
<img width="240" alt="Screen Shot 2020-12-22 at 2 28 29 PM" src="https://user-images.githubusercontent.com/2838819/102943322-8432a380-446c-11eb-877e-fde2cba7520d.png">
